hack/test-cmd.sh

```diff
@@ -308,7 +308,7 @@ osadm registry --create --credentials="${OPENSHIFTCONFIG}"
 # delete the registry resources
 osc delete dc docker-registry
 osc delete svc docker-registry
-osc delete rc docker-registry-1
+[ ! "$(osc get rc docker-registry-1)" ]
```

Comment:
Deleting the dc first deletes its replication controller and then deletes the config. Iff `osc delete dc docker-registry` fails to find the dc or fails to resize the replication controller then this will fail as well but these are corner cases are there are such in any command.